content/en/docs/concepts/services-networking/ingress.md

```diff
@@ -100,32 +84,63 @@ Each HTTP rule contains the following information:
 * An optional host. In this example, no host is specified, so the rule applies to all inbound
   HTTP traffic through the IP address specified. If a host is provided (for example,
   foo.bar.com), the rules apply to that host.
-* A list of paths (for example, `/testpath`), each of which has an associated backend defined with a `serviceName`
-  and `servicePort`. Both the host and path must match the content of an incoming request before the
+* A list of paths (for example, `/testpath`), each of which has an associated backend defined with a `service.Name`
+  and a `service.Port.Name` or `service.Port.Number`. Both the host and path must match the content of an incoming request before the
```

Comment:
Are we trying to say `service.name`, `service.port.name` or `service.port.number` instead?
Note the field names usually don't start with a capital letter.

Reply:
Good catch, thanks! I think we tend to confuse how we use them in Go with how most people will end up using them with YAML.